metaflow/flowspec.py

```diff
@@ -193,8 +193,11 @@ def _set_constants(self, graph, kwargs):
             setattr(self, var, val)
 
         # We store the DAG information as an artifact called _graph_info
+        for deco in flow_decorators(self):
+            if deco.name == "trigger":
+                for trig in deco.triggers:
+                    print(trig)
```

Comment:
is this for debugging purposes only? can be removed in that case.
same goes for the changes in `_vendor`, and what looks to be a local vscode config(?)